spec/collectionspace/mapper/data_mapper_core_spec.rb

```diff
@@ -94,13 +94,14 @@
           @fixture_doc = get_xml_fixture('core/loanin1.xml')
           @fixture_xpaths = test_xpaths(@fixture_doc, @handler.mapper[:mappings])
         end
-        it 'does not map unexpected fields' do
+        xit 'does not map unexpected fields' do
           diff = @mapped_xpaths - @fixture_xpaths 
           expect(diff).to eq([]) 
         end
 
         it 'maps as expected' do puts
-          @fixture_xpaths.each do |xpath| puts xpath
+          @fixture_xpaths.each do |xpath|
+            next if xpath == '/document/loansin_common/lenderGroupList/lenderGroup[2]/lender/text()'
```

Comment:
skip the other known bad xpath stemming from the issue described above, so it can run through and report any more problems (or succeed, letting us know there's nothing left to fix).

Reply:
Noted.